packages/plugins/src/plugins/re-rehearsal.plugin.ts

```diff
@@ -0,0 +1,48 @@
+import { isLineBreak } from 'typescript';
+
+import {
+  Plugin,
+  type PluginParams,
+  type PluginResult
+} from '@rehearsal/service';
+
+/**
+ * Removes all comments with `@rehearsal` tag inside
+ */
+export class ReRehearsalPlugin extends Plugin {
+  async run(params: PluginParams<undefined>): PluginResult {
+    const { fileName } = params;
+
+    const tag = '@rehearsal';
```

Comment:
This will be moved to plugin' options later - https://github.com/rehearsal-js/rehearsal-js/issues/314